CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,450 @@
+# 1 Test Objective
+SQL is an ANSI (American National Standards Institute) standard computer language, but there are still many different versions of SQL.However, in order to be compatible with ANSI standards, they must all support some major commands (such as SELECT, UPDATE, DELETE, INSERT, WHERE, and so on) in a similar way.  
+CUBRID SQL detection, SQL detection for each build to detect the basic functions of the database and ensure the correct output of commands.
+
+# 2 Test Deployments and Introduction
+To facilitate the execution of various test requirements, CUBRID QA has developed a range of automated tools. This chapter describes the environment construction.
+
+This is an implementation of SQL functions that we used to test CUBRID. 
+
+* CTP Tool  
+CTP is a testing tool for an open source project CUBRID. It is written in Java and easy to execute tests with a simple configuration
+Reference to CTP description：[CTP_README](https://github.com/CUBRID/cubrid-testtools/blob/develop/CTP/README.md)
+
+* Test Machines  
+
+  No|role|user|ip|hostname
+  :---:|:--:|:---:|:---:|:---:
+  1|Test node|sql1|192.168.1.76|func01
+  2|Test node|sql2|192.168.1.76|func01
+   3|Test node|sql3|192.168.1.76|func01
+  4|Test node|sqlbycci|192.168.1.76|func01
+  5|Test node|sql|192.168.1.77|func02
+  6|Test node|qa|192.168.1.161|winfunc01
+
+* Test Deployments  
```

Comment:
In Deployment chapter, split two introduction. One is General Deployment, another is Regression Test Deployment.

General Deployment:  suppose that the reader is one of developers.  CTP is used to execute check-in test. Please give him or her an installation guide.

Regression Test Deployment:  To introduce current regression test deployment for current QA.